ouroboros-consensus-cardano/test/cardano-test/Test/ThreadNet/Cardano.hs

```diff
@@ -73,7 +73,7 @@ import           Test.Util.TestEnv
 
 -- | Use 'MockCryptoCompatByron' so that bootstrap addresses and
 -- bootstrap witnesses are supported.
-type Crypto = MockCryptoCompatByron
+type Crypto = StandardCrypto
```

Comment:
The only difference between `MockCryptoCompatByron` and `StandardCrypto` is in the VRF and KES:

 - `MockKES` doesn't do any evolution.
 - `MockKES` and `MockVRF` are faster (see below).
 - The types involved in `MockKES` and `MockVRF` are potentially simpler to construct (though we mostly use `Arbitrary` instances that are defined for any `Crypto`).
 - `MockKES` and `MockVRF` yield more compact output in QuickCheck counterexamples.

A small benchmark with `hyperfine` yields that the Cardano ThreadNet is now ~50% slower: 

```bash
hyperfine -L crypto MockCryptoCompatByron,StandardCrypto './cardano-test-{crypto} -p "Cardano ThreadNet" --quickcheck-tests=2 --quickcheck-replay=0'
```

| Command | Mean [s] | Min [s] | Max [s] | Relative |
|:---|---:|---:|---:|---:|
| `MockCryptoCompatByron` | 6.319 ± 1.329 | 5.350 | 9.020 | 1.00 |
| `StandardCrypto` | 9.547 ± 2.617 | 7.489 | 15.095 | 1.51 ± 0.52 |

Overall CI time of the entire `cardano-test` suite didn't regress by the same factor as it does lots of other things (see [Hydra](https://ci.iog.io/job/IntersectMBO-ouroboros-consensus/main/x86_64-linux.native.haskell96.checks.ouroboros-consensus-cardano.cardano-test/all)), it seems to be longer by 20s from 5min20s to 5min40s.

---

Also, I don't think that the usage of `StandardCrypto` in ledger is absolutely necessary; it could be generalized to any `Crypto c` where just the hashes/signatures are constrained, but the VRF/KES is left unconstrained.

Given that larger changes in the crypto parameterization are planned[^1] I am inclined to leave the code as is for now.

[^1]: See https://github.com/IntersectMBO/cardano-ledger/issues/4223, which would make all ledger definitions crypto-monomorphic, but removing KES/VRF from `Crypto` to sth like `HeaderCrypto`, which would solve the issue at hand (as `MockCryptoCompatByron` and `StandardCrypto` only differ in header crypto).